ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -26,6 +26,9 @@ prefix_mask_packages: |
   # stick to GCC 9.x; using a too recent compiler in the compat layer complicates stuff in the software layer,
   # see for example https://github.com/EESSI/software-layer/issues/151
   >=sys-devel/gcc-10
+  # see https://github.com/EESSI/software-layer/issues/257
+  # and https://github.com/EESSI/software-layer/issues/258
+  <dev-libs/openssl-3
```

Comment:
You want `>=` here, see also https://github.com/trz42/compatibility-layer/pull/2

Reply:
Thanks. Should be fixed now. Resubmitting jobs.